denue/api/views.py

```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
+
+# Create your views here.
```

Comment:
Is this file needed?

Reply:
no, this file was automatically created, it has beed deleted  
commit: d455945648dd7f1cd703b07ea2a1254da4e67e6a